### ISSUE TITLE
why link a fork of my repo as the "original" mod lol

### ISSUE THREAD

**Issue by Coderx-Gamer** [gingerchicken/shulker-dupe-rewritten#9]

**Comment by gingerchicken** [comment]
My apologies, so allow me to explain:

Back when 1.18 first came out, I was in the duper trooper discord server and I updated your original mod. Now after I did that, everyone went insane saying that I had found a "new dupe" so duper trooper called me.

I explained that I legit just updated like one file and that it wasn't my dupe, we then parted ways.

A few days later he messages me back saying "hey can the fifth column add your mod to their client," and I thought well it isn't really my mod and in addition the original work doesn't have a license, so I will re-write it and put it under MIT.

I then, re-wrote it and created this repository, I didn't mean for it to be used by many people, I legit thought it would have just been used once by the fifth column.

With that in mind, as at the time you hadn't updated it or whatever, I linked my fork of the updated version and said that it was the "original" as that's what I was basing this project off.

I understand your frustration and I really respect you and your work. I didn't mean for this project to really mimic your achievement in any regard, I thought that credit was already placed as I linked an updated fork which linked directly to you.

I will add your name to the README and make sure that proper credit is given for any of your work in future.

Keep it up man!
